FacilityOntology.ttl

```diff
@@ -404,7 +404,7 @@ cco:MilitaryHeadquartersFacility rdf:type owl:Class ;
 ###  http://www.ontologyrepository.com/CommonCoreOntologies/Mine
 cco:Mine rdf:type owl:Class ;
          rdfs:subClassOf cco:Facility ;
-         cco:definition "A Facility that is designed to support the extraction of valuable minerals and geological materials from the earth from an orebody, lode, vein, seam, reef, or placer deposits which forms the mineralized package of economic interest to the miner."@en ;
+         cco:definition "A Facility that is designed to support the extraction of minerals or other geological materials from the earth from an orebody, lode, vein, seam, reef, or placer deposit which forms the mineralized package of interest to the miner."@en ;
```

Comment:
@cameronmore  Please consider simplifying to: "...from an orebody, lode, vein, seam, reef, or placer deposit within the earth."